HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+0.1.2 (2022-09-29)
```

Comment:
should please be released as I made bug fixes. We can chat about how to correctly use bumpversion -- I've never used it before

Reply:
Sure! One can pretty much follow the instructions given here: https://dask-ms.readthedocs.io/en/latest/contributing.html#deploying